packages/cluster-ui/README.md

```diff
@@ -17,3 +17,68 @@ export default props => (
   </div>
 );
 ```
+# Development Workflow with CRDB or CockroachCloud
+
```

Comment:
Currently yarn link is not working for CockroachCloud. Current workaround is to create tarball of cluster-ui manually 

`yarn build`
`yarn pack --prod`

and then install it in managed service project

`cd /console`
`yarn cache clean`
`yarn add --force /Users/<full path to cluster-ui>/cockroachlabs-cluster-ui-vXXXX.tgz`

and basically you need to do it manually after every change.

Reply:
Thx for documenting this. Added in separate section.